### ISSUE TITLE
Amélioration la section SGB

### ISSUE THREAD

**Issue by Olon11** [profcfuhrmanets/log210-enonce-lab1#91]
Le SGB a évolué et la section SGB de l'énoncé n'est plus à jour. Notamment :

- [ ] On ne devrait pas avoir le MDD du SGB dans l'énoncé. On doit plutôt faire référence au MDD dans le README du SGB.
- [ ] « Il possède une interface de configuration permettant de nettoyer le contenu correspondant aux notes ». Il n'y a aucune route du genre. Soit on en ajoute une, soit on retirer cette phrase.
- [ ] « L'API de SGB est disponible avec le code source. Lisez le README.md de SGB pour savoir comment générer cette documentation. ». Vrai, mais on peut aussi consulter la documentation directement en ligne. Je modifierais la phrase pour être plus générique et dire qu'on explique comment consulter la documentation dans le dépôt.

**Comment by fuhrmanator** [comment]
> On ne devrait pas avoir le MDD du SGB dans l'énoncé. On doit plutôt faire référence au MDD dans le README du SGB.

L'avantage d'avoir le MDD dans l'énoncé est que c'est concis. Je ne veux pas que SGB (et son code) soit à étudier profondément. SGB est censé être une boîte noire (système externe). 

OK pour les deux autres points.

**Comment by Olon11** [comment]
Dans ce cas, on peut mettre le lien vers le MDD du SGB dans l'énoncé plutôt que de dupliquer le fichier.

**Comment by Olon11** [comment]
Si l'image dans l'énoncé pointe vers le MDD dans le dépôt du SGB, on perd la partie SGA, mais on s'assure que le MDD est toujours à jour. À mon avis, la partie SGA n'apporte pas grande chose, donc ce n'est pas grave de l'enlever. Qu'est-ce que tu préfères?

**Comment by fuhrmanator** [comment]
Je préfère que tout soit dans SGA, y compris le MDD de SGB. Ce MDD décrit le problème que SGB résout (en partie). SGB doit être cohérent avec ça et pas dans l'autre sens.

**Comment by Olon11** [comment]
> Je préfère que tout soit dans SGA

Oui, j'avais compris cette partie. Je voulais savoir comment tu préférais procéder pour accomplir ça :

1. Le MDD du SGB est dans le dépôt du SGB. On met un lien vers ce MDD dans l'énoncé. On n'a plus la partie SGA dans le MDD.
2. Même chose que 1, mais on ajoute la partie SGA au MDD du SGB.
3. On met le MDD du SGB dans le dépôt de l'énoncé. On met un lien vers ce MDD dans le dépôt du SGB.

**Comment by fuhrmanator** [comment]
L'option 3 me va.
